core/src/column_names.py

```diff
@@ -0,0 +1,17 @@
+# orderbook column names
```

Comment:
The format of the names here is not consistent with the one used in Anduril. 
Respectively:
Market Timestamp, Gateway Timestamp, Sym, Market, BidSizes, BidPrices, OfferSizes, OfferPrices, Misc

Reply:
Yep, I know, I think it's better from a coding standard perspective so I have just renamed them into snake case